### DISCUSSION TITLE
Manpage styles are not loading

### DISCUSSION THREAD

**Discussion by yuriteixeira** [Homebrew/discussions#3009] [Everyday usage]
<img width="2032" alt="Screenshot 2022-02-23 at 08 47 13" src="https://user-images.githubusercontent.com/461055/155279274-559cb236-b792-4367-9688-c7bcd4ae6341.png">

**Comment by SMillerDev** [comment]
Yeah, there seem to be some issues with Github Pages, some links are also dead.

**Reply by yuriteixeira** [reply]
Indeed, like this one (which I was about to read :p) https://docs.brew.sh/brew/Cask-Cookbook
